### PR TITLE
Add enable en billing doc

### PR DESCRIPTION
**Pull request by Sherlock113**
Signed-off-by: Sherlock113 <sherlockxu@yunify.com>

### CODE REVIEW

File: content/en/docs/toolbox/metering-and-billing/enable-billing.md [hunk 1]
```diff
@@ -5,3 +5,59 @@ description: "Enable the billing function in KubeSphere to view the billing data
...
+   ```bash
+   kubectl edit cm kubesphere-config -n kubesphere-system
+   ```
+
+2. Add the retention day and price information under `metering` in the ConfigMap.
+
+   | Parameter                                  | Description                                                  |
+   | ------------------------------------------ | ------------------------------------------------------------ |
+   | `retentionDay`                             | `retentionDay` determines the date range displayed on the **Metering and Billing** page for users. The value of this parameter must be the same as the value of `retention` in [Prometheus](../../../faq/observability/monitoring/#how-to-change-the-monitoring-data-retention-period). |
+   | `currencyUnit`                             | The currency that displays on the **Metering and Billing** page. Currently allowed values are `CNY` (Renminbi) and `USD` (US dollars). If you specify other currencies, the console will display cost in CNY by default. |
```
Comment (yunkunrao) [inline content/en/docs/toolbox/metering-and-billing/enable-billing.md:24]: There is a type error in ZH doc. We should change "If you specify other currencies, the console will display cost in CNY by default." to "If you specify other currencies, the console will display cost in USD by default."

Comment (yunkunrao) [reply content/en/docs/toolbox/metering-and-billing/enable-billing.md:24]: This is related with https://github.com/kubesphere/website/pull/1407

Comment (Sherlock113) [reply content/en/docs/toolbox/metering-and-billing/enable-billing.md]: Fixed.

**Pull request merged**